pkgs/development/ocaml-modules/twt/default.nix

```diff
@@ -10,10 +10,9 @@ stdenv.mkDerivation {
 
   buildInputs = [ ocaml findlib ];
 
-  createFindlibDestdir = true;
-
-  configurePhase = ''
```

Comment:
I think we just need to `runHooks preConfigure` here and it should fix it.

This was broken by https://github.com/NixOS/nixpkgs/commit/a3ab43d3a45338f845a0f8a12ed34e84c8b79dae